src/Build/Graph/ProjectGraph.cs

```diff
@@ -79,8 +79,9 @@ public ProjectGraph(string entryProjectFile)
                 }
             }
 
+            entryProjectFile = FileUtilities.NormalizePath(entryProjectFile);
```

Comment:
Minor unrelated bug I found while testing this. Basically from the command line, the path given isn't absolute usually so the lookup below on line 87 was failing. Wasn't caught in the UT we have for this since the UT uses absolute paths when invoking MSBuild.exe.